### PR TITLE
Changing color of text around the app to black

### PR DESCRIPTION
**Pull request by elizzhao**
Lou requested we change the dominant color of text around the app to black so that it's uniform across different computer screens. 

Other changes: 
I darkened some of the gray text so that it doesn't look as stark next to the new black text. 
I changed the fontWeight on buttons to match the mocks on InVision.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -18,7 +18,7 @@ html, body {
 body {
   background-color: #ededed;
   margin: 0;
-  color: #4a4a4a;
+  color: #000000;
```
Comment (zarsky-broad) [inline src/style.css]: This shouldn't have to be set.

File: src/components/common.js [hunk 2]
```diff
@@ -110,7 +112,7 @@ export const MenuButton = ({ disabled, children, ...props }) => {
     style: {
       display: 'flex', alignItems: 'center',
       fontSize: 12, minWidth: 125, height: '2rem',
-      color: disabled ? colors.gray[2] : colors.gray[0],
+      color: disabled ? colors.gray[2] : 'black',
```
Comment (zarsky-broad) [inline src/components/common.js]: I'd vote for `undefined` instead of `'black'`

File: src/components/TooltipTrigger.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ const styles = {
     transformOrigin: 'bottom'
   },
   lightBox: {
-    background: 'white', color: colors.gray[0],
+    background: 'white', color: 'black',
```
Comment (zarsky-broad) [inline src/components/TooltipTrigger.js]: This shouldn't have to be defined? You should def double check, though.

**Review by zarsky-broad — APPROVED** [review]
We should check with Lou on font weights; he tends to use whatever in the wires, but we now have a total of 3 font weights in the app, and I thought we just wanted 2.

**Review by katevoss — APPROVED** [review]

**Comment by elizzhao** [comment]
Checked with Lou about font weights. 
He added a 'bold' fontWeight that is now in many different locations around the app. Committed the changes that match bold fontWeights to the mocks.

**Pull request merged**